rskj-core/src/main/java/co/rsk/peg/whitelist/WhitelistResponseCode.java

```diff
@@ -0,0 +1,19 @@
+package co.rsk.peg.whitelist;
+
+public enum WhitelistResponseCode {
+    GENERIC_ERROR(-10),
+    INVALID_ADDRESS_FORMAT_ERROR(-2),
+    ALREADY_EXISTS_ERROR(-1),
```

Comment:
```suggestion
    ADDRESS_ALREADY_WHITELISTED(-1),
```
Sounds a bit more descriptive I think

Reply:
Agree.